src/utils/configs.js

```diff
@@ -28,6 +27,8 @@ let isAdmin = false;
   }
 });
 
+configs.IS_LOCAL_OR_CUSTOM_CLIENT = document.querySelector("meta[name='env:thumbnail_server']") ? false : true;
```

Comment:
Same comment above. Generally not great to use an unrelated env config as a proxy. A "why" comment would be nice here too.